src/Rector/Architecture/DependencyInjection/ReplaceVariableByPropertyFetchRector.php

```diff
@@ -91,8 +91,6 @@ public function default()
 
     public function isCandidate(Node $node): bool
     {
-        $this->activeVariableInfo = null;
```

Comment:
Why?

Reply:
Isn’t necessary, we’ve already initialized the property when declared it

Reply:
It changed on each iteration, it's not one-time action.

Often it's not null.

Reply:
So we should change the implementation. Can you handle it?

Reply:
Why would you change it? It works great now

Reply:
So let's keep it. Maybe we should remove PHPStan in the future than :)

Reply:
Why?